test/DisplayCervicalCancerMedicalHistory/cases/BiopsyResultsOnlyObservationHasProperCode.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Biopsy Results Only Observation Has Proper Code
+
+data:
+-
+  resourceType: Patient
+  name: Joanne Smith
+  gender: female
+  birthDate: 1980-01-01
+  extension:
+  -
+    url: http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex
+    valueCode: F
+-
+  resourceType: DiagnosticReport
+  code: urn:oid:1.2.840.114350.1.13.284.3.7.5.737384.95#SURG01
+  status: final
+  effectiveDateTime: 2021-05-08T19:46:00Z
+  issued: 2021-05-10T16:42:59Z
+- 
+  resourceType: Observation
+  code: LOINC#65753-6 Cervix Pathology biopsy report
+  status: final
+  valueCodeableConcept: SNOMEDCT#285836003 CIN 1
+  effectiveDateTime: 2021-05-08T19:46:00Z
+  issued: 2021-05-10T16:42:59Z
+
+results:
+  HistologyDiagnosticReports: $should have length 1
+  PertinentObservationsSummary: $should have length 0
```

Comment:
I don't fully understand the definition of `PertinentObservationsSummary`, if an Observation with biopsy is feed, shouldn't it be added as PertinentObservations and count 1?

Reply:
The naming here gets slightly confusing. But the answer is related to the answer to the comment above. An Observation on its own is not typically used to represent a lab test. The lab test is represented by the DiagnosticReport resource. The DiagnosticReport can then reference Observations that contain more detailed information about the lab test, such as the results, or in this specific case, the actual LOINC code of the biopsy test itself.
	
The [PertinentObservationsSummary](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/DashboardLibrary.cql#L597) CQL expression unions together other relevant Observations that our CDS looks for - clinical elements that are represented specifically by an Observation. It wouldn't be _incorrect_ to call the biopsy Observations "pertinent" and list them within this field. But then we would risk showing duplicate information in the Dashbaord, since the biopsy test would show up in both the DiagnosticReportsSummary and the PertinentObservationsSummary.

Reply:
Oh, that was the definition of "pertinent". Thanks for clarifying.